### PR TITLE
Einstein Engines Cybernetic Limbs

### PR DESCRIPTION
**Pull request by Aikakakah**
## About the PR
Imported the cybernetic limbs from EE. 

## Why / Balance
I'm sure there's plenty of players who would appreciate finally being able to give their characters a robotic limb! 

## Technical details
Adjusted species files to increase marking limits, added both variations of the EE cybernetic limbs.

## Media
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: Aikakakah
- add: Added cybernetic limbs!

### CODE REVIEW

File: Resources/Prototypes/Einstein-Engines/Entities/Mobs/Customization/cyberlimbs/bishop.yml
Comment (MilonPL) [inline Resources/Prototypes/Einstein-Engines/Entities/Mobs/Customization/cyberlimbs/bishop.yml:1]: Dwarf, HumanoidFoxes unnecessary all throughout the file.

File: Resources/Prototypes/Einstein-Engines/Entities/Mobs/Customization/cyberlimbs/hesphiastos.yml
Comment (MilonPL) [inline Resources/Prototypes/Einstein-Engines/Entities/Mobs/Customization/cyberlimbs/hesphiastos.yml:1]: Dwarf, HumanoidFoxes unnecessary all throughout the file.

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Milon, god's strongest YML warrior 😸 

Seems pretty normal as YAML, didn't catch anything wrong. I would approve, but I'm gonna pull Game Directors so they look at this because it incorporates corporations from 13 into the game via the Names :tm: and also to, if nothing else, have a third pair of eyes on it. I hate to be that fuckin' nerd but I gotta be.

**Comment by TadJohnson00** [comment]
Mixed on this, will get back to you shortly.

**Comment by Astrumif-y** [comment]
Definitely a must-have for delta , ive always wanted to give a character or two a robot limb after an important moment that would justify such a thing

**Comment by TadJohnson00** [comment]
Alright, yep, I'll be fine with approving this if you remove the company names from the locale and replace it with numbers (right robotic arm 1, etc.)

**Review by ghost — APPROVED** [review]
Looks fine with the generic-ization of these, now.

**Review by MilonPL — APPROVED** [review]

**Pull request merged**